src/configs/typescript.ts

```diff
@@ -166,7 +167,10 @@ export async function typescript(
           files: filesTypeAware,
           ignores: ignoresTypeAware,
           name: 'antfu/typescript/rules-type-aware',
-          rules: typeAwareRules,
+          rules: {
+            ...typeAwareRules,
+            ...objectPick(overrides, Object.keys(typeAwareRules)),
```

Comment:
I am not so sure about this approach, it feels to implicit. Maybe we could just have an additional `overridesTypeAware` option?

Reply:
Nice idea